devel/external/AEGIS/AEGIS_25.2_training_sample_readme.txt

```diff
@@ -0,0 +1,109 @@
+This README file describes the AEGIS F814W<25.2 training sample, which is a
+randomly ordered sample of galaxies in the AEGIS field with AB magnitude<25.2, in F606W(V) and F814W(I) bands.
+
+NOTE: The columns in the catalog are those expected by galsim.RealGalaxyCatalog.
+However, some parts of that analysis, i.e parametric fits, were not performed.
+Hence those values were manually set and do not have physical meaning.
+
+(1) The catalogs themselves are in AEGIS_galaxy_catalog_[VI]_25.2.fits (where
+[VI] indicates either V or I).  These files can be used to initialize a
+galsim.RealGalaxyCatalog or galsim.COSMOSCatalog instance, and contain a FITS
+tables that could also be  read in directly using, e.g., python or IDL.  The
+fields that are included for each galaxy are:
+
+IDENT: AEGIS object identifier. This value is internally derived.  Therefore
+do not rely on this IDENT for cross-matching between other AEGIS catalogs.
+
+RA, DEC: Right ascension and declination (J2000, degrees).  While this is not
+needed for image simulation, it is necessary for proper cross-matching with
+other catalogs.
+
+MAG: magnitude (MAG_AUTO calculated from sextractor).
+
+BAND: Passband for the observed image (F814W/F606W).
+
+WEIGHT: A weight factor to use to account for the fact that larger galaxies are
+preferentially excluded from this catalog due to the proximity to CCD edges.
+(Currently all weights are set to 1.0).
+
+GAL_FILENAME: Name of the file containing the galaxy image.  This is just a
+filename, not including a full path.
+
+PSF_FILENAME: Name of the file containing the PSF image for this galaxy.
+This is just a filename, not including a full path.
+
+GAL_HDU: HDU containing the image of this galaxy in the file called GAL_FILENAME.
+
+PSF_HDU: HDU containing the image of the PSF for this galaxy in the file called PSF_FILENAME.
+
+PIXEL_SCALE: pixel scale for the galaxy and PSF images in arcsec (0.03 for all bands).
+
+NOISE_MEAN: mean value of the pixels in the image that do not contain the galaxy.
+
+NOISE_VARIANCE: variance of the pixel values for pixels in the image that do not contain the galaxy.
+
+
+(2) The files containing galaxy and PSF images are
+AEGIS_galaxy_images_[VI]_25.2_n?.fits and AEGIS_galaxy_PSF_images_[VI]_25.2_n?.fits. The number that goes in place of the "?" ranges from 0 to 29.
+
+
+(3) *Parametric fits have not been performed and the values in the table were manually set.*
```

Comment:
If you don't plan to make parametric fits and only intend for people to use this as a RealGalaxyCatalog, then why make a file for parametric fits at all?  If you want to use a RealGalaxyCatalog (or a COSMOSCatalog without parametric fits enabled) you don't need this file, and it seems better to omit it than to have a fake one.